packages/app/src/components/nominationBanner/resendEmailModal.js

```diff
@@ -0,0 +1,149 @@
+        <div className="email-modal-container">
+          <button
+            className="exit-button"
+            onClick={props.toggleEmailModalState}
+          >
+            &times;
+          </button>
+          <form className="email-form" onSubmit={handleSubmit}>
+            <div className="email-form-container">
+              <fieldset className="resend-fieldset">
+                <legend className="resend-legend">Recipient</legend>
+                <div>
+                  <label htmlFor="family-member" className="survey">
+                    <input
+                      type="radio"
+                      value="family-member"
+                      id="family-member"
+                      checked={recipientChecked === 'family-member'}
+                      onChange={handleRecipientChange}
+                      className="family-member radio"
+                    />
+                    Family Member
+                  </label>
+                </div>
+                <div>
+                  <label
+                    htmlFor="healthcare-provider"
+                    className="survey"
+                  >
+                    <input
+                      type="radio"
+                      value="healthcare-provider"
+                      id="healthcare-provider"
+                      checked={
+                        recipientChecked === 'healthcare-provider'
+                      }
+                      onChange={handleRecipientChange}
+                      className="healthcare-provider radio"
+                    />
+                    Healthcare Provider
+                  </label>
+                </div>
+              </fieldset>
+
+              <fieldset className="resend-fieldset">
+                <legend className="email-legend resend-legend">
+                  Email Type
+                </legend>
+                {
+                  (props.status === 'Awaiting HIPAA' || props.status === 'HIPAA Verified') ? 
+                  (   <div>
+                    <label htmlFor="hipaa" className="survey">
+                      <input
+                        type="radio"
+                        value="hipaa"
+                        id="hipaa"
+                        checked={emailTypeChecked === 'hipaa'}
+                        onChange={handleEmailTypeChange}
+                        className="hipaa radio"
+                      />
+                      HIPAA
+                    </label>
+                  </div>) : (
+                    <div>
+                      <div>
+                    <label htmlFor="hipaa" className="survey">
+                      <input
+                        type="radio"
+                        value="hipaa"
+                        id="hipaa"
+                        checked={emailTypeChecked === 'hipaa'}
+                        onChange={handleEmailTypeChange}
+                        className="hipaa radio"
+                      />
```

Comment:
Seems like this section is duplicated between here and line 81. Might be a way to clean that up a bit.